test/datetime.t.cpp

```diff
@@ -973,6 +973,11 @@ int main (int, char**)
     t.is (later.day (),    30, "later -> d = 30");
     t.is (later.year (), 9999, "later -> y = 9999");
 
+    Datetime earlier ("earlier");
+    t.is (earlier.month (),   1, "earlier -> m = 1");
+    t.is (earlier.day (),     1, "earlier -> d = 1");
+    t.is (earlier.year (), 1970, "earlier -> y = 1970");
+
```

Comment:
As far as I see it, a test for "before" is missing. Something like
```
Datetime before ("before");
t.is (before.month (),   1, "before -> m = 1");
t.is (before.day (),     1, "before -> d = 1");
t.is (before.year (), 1970, "before -> y = 1970");
```
Yes, "before" is mapped to "earlier" in `Datetime.cpp:1626`, but without this, you only verify "before" *is* mapped,  not whether it is mapped correclty.

Reply:
Done, I am testing for `before == earlier`, and also added the same test for `someday == later`.